language/tools/move-cli/tests/testsuite/doctor_with_stdlib/src/modules/M.move

```diff
@@ -0,0 +1,10 @@
+address 0x2 {
+module M {
+    use 0x1::Debug;
+    struct Coin { f: u64, g: u64 }
+
+    fun dangle_after_move(_c: Coin) {
```

Comment:
Nit: `f` or some other name? I think this silly name came from my repro, which was not very well minified :)